src/constants/options/post.js

```diff
@@ -1,3 +1,3 @@
-export default ['COPY LINK', 'REBLOG', 'REPLY'];
+export default ['COPY LINK', 'REBLOG', 'REPLY', 'SHARE'];
 
-export const VALUE = ['copy', 'reblog', 'reply'];
+export const VALUE = ['copy', 'reblog', 'reply', 'share'];
```

Comment:
I think we should move to language files

Reply:
updated for language